emulator.c

```diff
@@ -333,6 +367,10 @@ execute_instruction(i8080 *cpu, uint8_t opcode)
       }
     case 0xd3: // NOLINT
       printf("OUT ");
+      // cpu->a will be byte one and port to write to is port 2
+      uint8_t port = cpu_read_mem(cpu, cpu->pc + 1);
+      printf("%u", port);
+      cpu->pc += 2;
```

Comment:
Same comment as 0xc6 regarding incrementing pc, think you just to need to increment by 1 here.